src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/NbWorkerTopology.java

```diff
@@ -239,45 +260,58 @@ public StormTopology createTopology() {
                 .shuffleGrouping(SWITCHES_BOLT_NAME, StreamType.DISCO.toString())
                 .shuffleGrouping(FEATURE_TOGGLES_BOLT_NAME, FeatureTogglesBolt.STREAM_NOTIFICATION_ID);
 
-        KafkaBolt kafkaNbBolt = buildKafkaBolt(topologyConfig.getKafkaNorthboundTopic());
+        KafkaBolt kafkaNbBolt = buildKafkaBolt(topologyConfig.getKafkaNorthboundTopic(),
+                getZkTopoName(), getConfig().getBlueGreenMode());
```

Comment:
Does it make sense to introduce a dedicated "build" method instead of repeating "getZkTopoName(), getConfig().getBlueGreenMode()" each time?